descwl_shear_sims/stars.py

```diff
@@ -77,34 +77,31 @@ def get_objlist(self, *, survey, noise):
 
         Returns
         -------
-        [galsim objects]
+        [galsim objects], [shifts],
+        [bright_objlist], [bright_shifts], [bright_mags]
```

Comment:
doc string wrong?

Reply:
Looks right to me

Reply:
ohh woops I only read the last line! sorry!